src/fiatconnect/saga.test.ts

```diff
@@ -922,4 +893,77 @@ describe('Fiatconnect saga', () => {
       )
     })
   })
+
+  describe('_getSpecificQuote', () => {
+    it('fetches and returns a quote if fiat account type and provider ID matches', async () => {
+      await expectSaga(_getSpecificQuote, {
+        digitalAsset: CiCoCurrency.CUSD,
+        cryptoAmount: 2,
+        flow: CICOFlow.CashOut,
+        providerId: 'provider-two',
+        fiatAccountType: FiatAccountType.BankAccount,
+      })
+        .provide([
+          [
+            call(_getQuotes, {
+              flow: CICOFlow.CashOut,
+              digitalAsset: CiCoCurrency.CUSD,
+              cryptoAmount: 2,
+              providerIds: ['provider-two'],
+            }),
+            [mockFiatConnectQuotes[1]],
+          ],
+        ])
+        .returns(normalizeFiatConnectQuotes(CICOFlow.CashOut, [mockFiatConnectQuotes[1]])[0])
+        .run()
+    })
+    it('throws if no quote with matching fiatAccountType is found', async () => {
+      await expect(
+        async () =>
+          await expectSaga(_getSpecificQuote, {
+            digitalAsset: CiCoCurrency.CUSD,
+            cryptoAmount: 2,
+            flow: CICOFlow.CashOut,
+            providerId: 'provider-two',
+            fiatAccountType: FiatAccountType.DuniaWallet,
+          })
+            .provide([
+              [
+                call(_getQuotes, {
+                  flow: CICOFlow.CashOut,
+                  digitalAsset: CiCoCurrency.CUSD,
+                  cryptoAmount: 2,
+                  providerIds: ['provider-two'],
+                }),
+                [mockFiatConnectQuotes[1]],
+              ],
+            ])
+            .run()
+      ).rejects.toThrow('Could not find quote')
+    })
+    it('throws if no quote with matching provider ID is found', async () => {
```

Comment:
@jh2oman is this even a valid scenario? This tests for the 2nd condition here https://github.com/valora-inc/wallet/blob/806191e7ca28d5ad81b5d1f95bf400f10770dfd4/src/fiatconnect/saga.ts#L328, but can that even happen since we get quote for the specific provider? Wonder if we can drop it.

Reply:
Shouldn't be possible, I think. We have [a check](https://github.com/valora-inc/valora-rest-api/blob/main/src/fiatConnect/getQuotes.ts#L37) in valora-rest-api that should prevent this from ever occuring.

Reply:
Yea shouldn't be possible. I think thats a relic of one iteration of this code where it may have been possible, where we got ALL quotes first or something like that. 

I think we can remove

Reply:
👍 removed